libs/Zend/Session/Exception.php

```diff
@@ -55,7 +55,10 @@ class Zend_Session_Exception extends Zend_Exception
      */
     static public function handleSessionStartError($errno, $errstr, $errfile, $errline, $errcontext)
     {
-        self::$sessionStartError = $errfile . '(Line:' . $errline . '): Error #' . $errno . ' ' . $errstr;
+        if (!isset(self::$sessionStartError)) {
+            self::$sessionStartError = '';
+        }
+        self::$sessionStartError .= $errfile . '(Line:' . $errline . '): Error #' . $errno . ' ' . $errstr;
```

Comment:
Might be good to separate the errors by newlines/spaces for readability? Otherwise LGTM.